docs/getting-started/create-an-app.md

```diff
@@ -28,7 +28,7 @@ don't worry about the choice, it's easy to change later! Here is a
 The easiest way to run the create app package is with `npx`:
 
 ```bash
-npx @backstage/create-app
+npx @backstage/create-app@latest
```

Comment:
.... erm, is this necessary? i usually leave it off, and i believe that'll be the default, no?

https://www.npmjs.com/package/@backstage/create-app?activeTab=versions

Since it's controlled by Backstage the project, and we know the project always sets this to the default, I feel unnecessary.

for new users, i'd default to as simple as possible.

Alternatively, and this could be just my lack of expertise, the npm docs do say leaving it off just always installs the latest version. is the concern it might accidentally pick up the nightly?  if so, then yeah good with `@latest` :)

Reply:
So in most cases it should be fine without but there's also cases where this gets cached locally and causes problems for people. As this is likely the first way you interact with Backstage it's important that you got a solid experience so this errors on the side of caution. This PR added the `@latest` to most places already: https://github.com/backstage/backstage/pull/15412. Here's an example of where not having it caused issues: https://github.com/backstage/backstage/issues/16311